pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -717,6 +717,28 @@ def test_agg_relabel_multiindex_duplicates():
     tm.assert_frame_equal(result, expected)
 
 
+def test_groupby_aggregate_empty_key():
+    # GH: 32580
+    df = pd.DataFrame({"a": [1, 1, 2], "b": [1, 2, 3], "c": [1, 2, 4]})
+    result = df.groupby("a").agg({"c": ["min"]})
+    result_2 = df.groupby("a").agg({"b": [], "c": ["min"]})
+    expected = pd.DataFrame(
+        [1, 4],
+        index=pd.Index([1, 2], dtype="int64", name="a"),
+        columns=pd.MultiIndex.from_tuples([["c", "min"]]),
+    )
+    tm.assert_frame_equal(result, expected)
+    tm.assert_frame_equal(result, result_2)
+
+
+def test_groupby_aggregate_empty_key_empty_return():
+    # GH: 32580 Check if everything works, when return is empty
```

Comment:
you might be able to include this case in the above parmaterization as well, if too many gymnastics, then ok to have this test separate

Reply:
I think the parametrization would be more complex and longer than the second test case, because we would have to parametrize the whole expected DataFrame. 

I would like to avoid that